libsplinter/src/admin/store/diesel/migrations/postgres/migrations/2020-08-18-213009_admin_service_store/up.sql

```diff
@@ -73,7 +73,6 @@ CREATE TABLE IF NOT EXISTS proposed_service_argument (
     key                       TEXT NOT NULL,
     value                     TEXT NOT NULL,
     PRIMARY KEY (circuit_id, service_id, key),
-    FOREIGN KEY (service_id) REFERENCES proposed_service(service_id),
```

Comment:
Seems to be a typo in the commit message: "... however this is not
service_id is not ..."

Reply:
Fixed